cw/capture_otbn_vertical_batch.yaml

```diff
@@ -0,0 +1,37 @@
+device:
+  fpga_bitstream: objs/lowrisc_systems_chip_earlgrey_cw310_0.1_ecdsa.bit
+  fw_bin: objs/ecc256_keygen_serial_fpga_cw310.bin
+  # Target operating frequency
+  pll_frequency: 100000000
+  baudrate: 115200
+capture:
+  # Currently, 'p256' is the only supported curve.
+  curve: p256
+  # These byte-lengths are passed to the chipwhisperer key/plaintext generator.
+  # Since this is not encryption, they don't correspond to key/plaintext, but
+  # rather seed/mask.
+  use_fixed_key_iter: False # See https://github.com/lowRISC/ot-sca/issues/116
+  key_len_bytes: 40
+  plain_text_len_bytes: 40
+  masks_off: false
+  # Output length (most likely unused).
+  output_len_bytes: 40
+  # Samples per trace
+  # Fifo Size is 131070 = max num samples to avoid multiple runs per trace
+  num_samples: 3500
+  # Offset in samples
+  offset: 0
+  # scope gain in db
+  scope_gain: 24
+  num_traces: 40000
+  batch_prng_seed: 6
```

Comment:
Can we just add `batch_prng_seed:` option to `capture_otbn_vertical.yaml` instead of creating a new configuration file?

Reply:
Sure I can add that. My thought was, because the PLL frequency and number of samples need to be different than in non-batch mode (for non-batch mode 100 MHz is to fast), to add a separate config file for batch mode. Also in batch mode 100 Mhz is hardcoded into cw_segmented.py (see https://github.com/lowRISC/ot-sca/issues/129, @m-temp explained it there). Hence, I thought it might be a good idea to have a separate config file, to be able to run batch mode "out of the box" without having to change capture parameters.

Reply:
I deleted the two additional configuration files, and inserted `batch_prng_seed` into the existing one. Additionally, I added a comment regarding the PLL frequency, that for non-batch mode max. 91 MHz is possible.